docs/training.md

```diff
@@ -0,0 +1,204 @@
+# Training
+
+The prebuilt models will always be improved by adding data from the target area. In our work, we have found that even one hour's worth of carefully chosen hand-annotation can yield enormous improvements in accuracy and precision.
+We envision that for the majority of scientific applications at least some fine-tuning of the prebuilt model will be worthwhile. When starting from the prebuilt model for training, we have found that 5-10 epochs is sufficient. 
```

Comment:
We expect that the prebuilt model will benefit from at least some fine-tuning for the vast majority of scientific applications. We have discovered that 5–10 epochs of training with the prebuilt model are adequate.
The improvement of a retraining task after 10–30 epochs has never been observed, but it is theoretically feasible if there are very big datasets with extremely varied classes.